runtime/JavaScript/package.json

```diff
@@ -51,14 +56,18 @@
   },
   "exports": {
     ".": {
+      "types": "./src/antlr4/index.d.cts",
+      "import": "./dist/antlr4.node.mjs",
+      "require": "./dist/antlr4.node.cjs",
+      "default": "./dist/antlr4.node.mjs",
       "node": {
-        "types": "./src/antlr4/index.d.ts",
+        "types": "./src/antlr4/index.d.cts",
         "import": "./dist/antlr4.node.mjs",
         "require": "./dist/antlr4.node.cjs",
         "default": "./dist/antlr4.node.mjs"
       },
       "browser": {
-        "types": "./src/antlr4/index.d.ts",
+        "types": "./src/antlr4/index.d.cts",
```

Comment:
I believe this should be removed ?

Reply:
Yes, removed it and tested with different module resolutions like `NodeNext`, `Bundler` and `Node`. Seems working.